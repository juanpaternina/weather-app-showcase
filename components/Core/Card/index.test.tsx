import { render } from '@testing-library/react-native';

const { Card } = require('.');

describe('Card', () => {
  test('It should render a View component', () => {
    const { toJSON } = render(<Card />);
    expect(toJSON()).toMatchSnapshot();
  });
});
