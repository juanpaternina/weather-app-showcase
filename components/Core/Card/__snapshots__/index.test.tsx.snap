// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card It should render a View component 1`] = `
<View
  style={
    {
      "backgroundColor": "black",
      "borderBottomLeftRadius": 10,
      "borderBottomRightRadius": 10,
      "borderTopLeftRadius": 10,
      "borderTopRightRadius": 10,
      "minHeight": 10,
    }
  }
/>
`;
